APIClient/Forecast.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  Weather.swift
+//  APIClient
+//
+//  Created by Colin Otchere on 05.06.16.
+//  Copyright © 2016 iOS Dev Kurs Universität Heidelberg. All rights reserved.
+//
+
+import Foundation
+import Freddy
+import UIKit
+
+struct WeatherRawData:JSONDecodable{
+    
+    let currentWeather: Weather
+    let dailyWeather: Data
+    let timezone: String
+    
+    init(json: JSON) throws {
+        self.timezone = try json.string("timezone")
+        self.currentWeather = try json.decode("currently")
+        self.dailyWeather = try json.decode("daily")
+    }
+}
+
+struct Data: JSONDecodable{
+    
+    let data: [WeatherData]
+    
+    init(json: JSON) throws {
+        self.data = try json.arrayOf("data")
+    }
+}
+struct Weather:JSONDecodable{
+        
+    let time: Int
+    let summary: String
+    let icon: String
+    var temperatur: Double
+    var summaryGer: String
+    
+    init(json: JSON) throws {
+            
+        self.summary = try json.string("summary")
+        self.time = try json.int("time")
+        self.icon = try json.string("icon")
+        self.temperatur = try json.double("temperature")
+        self.summaryGer = ""
+        convertToCelsius()
+        self.summaryGer = translate(self.icon)
+    }
+
+    mutating func convertToCelsius(){
```

Comment:
Hier verlässt du dich darauf, dass diese Funktion genau einmal aufgerufen wird. Speicher die Temperatur lieber direkt in einer bestimmten Einheit.